### ISSUE TITLE
Apple Pay payment sheet won't be presented 

### ISSUE THREAD

**Issue by maysamsh** [stripe/stripe-ios#1783]
## Summary

I'm using the built-in UI to select/add/change payment method within my app (following https://stripe.com/docs/mobile/ios/basic). When user selects a card everything works fine and they can proceed with the payment but when Apple Pay is selected (I enabled Apple Pay) nothing happens after calling `paymentContext?.requestPayment()` and when it's being called with Apple Pay as the payment option calling `paymentContext?.presentPaymentOptionsViewController()` does not show `STPPaymentOptionsViewController` anymore while `paymentContext` is not nil. 

## iOS version
iOS 14.4.2

## Installation method
Cocoapods

## SDK version
21.3.1

There is no error or message in the console to help me to find out what's going on. 

Should I follow this https://stripe.com/docs/apple-pay when Apple Pay is selected?

**Comment by aliriaz-stripe** [comment]
@maysamsh hello, just to make sure I have this right: 

1/ You call `paymentContext.presentPaymentOptionsViewController()` and select "Apple Pay"
2/ back on your screen, you call `paymentContext.requestPayment()` 

after step 2, you aren't seeing anything happen? 

a) Can you send over a reproduction video and any logs from Xcode at that time? 

b) I assume you have done the pre-requisite steps for Apple Pay, is that right?

**Comment by davidme-stripe** [comment]
Closing this out as we haven't heard back in a while — please let us know if you're still experiencing this issue.

**Comment by mitchtreece** [comment]
@alicee-stripe This is happening for me as well after updating to 21.5. Confirming your steps listed above:

1. The payment options vc is presented, and "Apple Pay" is selected
2. Payment option page is dismissed, and `requestPayment()` is called
3. Apple Pay sheet is **not** presented, but parent still goes through via another card on file

**Comment by maysamsh** [comment]
I implemented the other method for when Apple Pay  is selected

**Comment by mitchtreece** [comment]
@maysamsh What other method?

**Comment by maysamsh** [comment]
The method to have Apple Pay alone

**Comment by mitchtreece** [comment]
@maysamsh Yes, would you mind telling me what that function is?
